### PR TITLE
issue-4843-style-collectionlist/create-style-enhancement

### PR DESCRIPTION
**Pull request by rebmullin**
this is a potential "fix" /start to this enhancement requested in #4843 (collections list create style enhancement)

note: 
copy here is a placeholder / tbd

regarding styles - I have very basic styles here  - please let me know if this is ok/suggestions:

desktop:
collection placeholder text-desktop

mobile:
collection placeholder text-mobile

@bobsilverberg  @pwalm @muffinresearch

### CODE REVIEW

**Comment by bobsilverberg** [comment]
Thanks for this, and all the other PRs too, @rebmullin. I'm traveling back to Canada tomorrow, and if I'm able to get online at the airport I'll try to review some of these, but it may be that I cannot get to them until next week. Just wanted to let you know.

**Comment by rebmullin** [comment]
hey @bobsilverberg  - no worries at all but thanks for letting me know : )  we'll catch up next week!

File: src/amo/components/CollectionList/index.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ export class CollectionListBase extends React.Component<Props> {
           <Card className="CollectionList-create">
             {this.renderManager()}
           </Card>
+          <p className="CollectionList-placeholder">
+            Please save your collection and then you can add add-ons to it
```
Comment (bobsilverberg) [inline src/amo/components/CollectionList/index.js]: We need to make sure that all strings that are displayed are localizable, and to do that we need to wrap it in an `i18n` function. In this case we can use `i18n.gettext`. You can see an example of this in the same file at line 38 where we have `logInText={i18n.gettext('Log in to create a collection')}`.

File: src/amo/components/CollectionList/styles.scss [hunk 2]
```diff
@@ -20,3 +20,13 @@
     margin: 10px 0 0;
   }
 }
+
+.CollectionList-placeholder {
+  margin: 50px auto;
+  opacity: 0.75;
```
Comment (bobsilverberg) [inline src/amo/components/CollectionList/styles.scss]: I don't see us setting anything else to an `opacity` of `0.75`. Why did you choose to do that? I'm not suggesting it's wrong, just wondering about the motivation to do it. We should also pass this by @pwalm to see what he thinks. He's one of our UX people and generally makes the decisions about how the site should look.

File: src/amo/components/CollectionList/index.js (hunk 1 above)
Comment (rebmullin) [reply src/amo/components/CollectionList/index.js]: ah! got it - thanks

File: src/amo/components/CollectionList/styles.scss (hunk 2 above)
Comment (rebmullin) [reply src/amo/components/CollectionList/styles.scss]: i guess when i think "placeholder" text i think lower opacity/lighter text color i guess : ) i can def remove that i just figured styles these were temp styels and were going to be adjusted anyway.

Comment (bobsilverberg) [reply src/amo/components/CollectionList/styles.scss]: Fair enough, that certainly makes sense. Let's see what @pwalm thinks about it.

**Comment by rebmullin** [comment]
@bobsilverberg  - changes made so text is now localizable -please take another look?

File: src/amo/components/CollectionList/index.js [hunk 3]
```diff
@@ -45,12 +45,17 @@ export class CollectionListBase extends React.Component<Props> {
...
+    const { i18n } = this.props;
+
     return (
       <div className="CollectionList">
         <div className="CollectionList-wrapper">
           <Card className="CollectionList-create">
             {this.renderManager()}
           </Card>
+          <p className="CollectionList-placeholder">
+            {i18n.gettext('Please save your collection and then you can add add-ons to it')}
```
[the remark this answers is not in the document]
Comment (rebmullin) [reply src/amo/components/CollectionList/index.js]: sure thing

**Comment by rebmullin** [comment]
@bobsilverberg -  thanks for the feedback. I've updated the line length.  

Sure once we get approval on style approval i'll open new issue to track copy

**Comment by bobsilverberg** [comment]
@rebmullin This is good to land as is. Could you please open a new issue to track updating this with the new copy?

**Comment by rebmullin** [comment]
yep will do- thanks @bobsilverberg

**Pull request merged**